src/quo/components/tabs/tabs/view.cljs

```diff
@@ -84,36 +86,20 @@
      :on-press            #(on-press % index)}
     label]])
 
-(defn view
-  " Common options (for scrollable and non-scrollable tabs):
-
-  - `blur?` Boolean passed down to `quo.components.tabs.tab/tab`.
-  - `data` Vector of tab items.
-  - `on-change` Callback called after a tab is selected.
-  - `size` 32/24
-  - `style` Style map passed to View wrapping tabs or to the FlatList when tabs
-    are scrollable.
-
-  Options for scrollable tabs:
-  - `fade-end-percentage` Percentage where fading starts relative to the total
-    layout width of the `flat-list` data.
-  - `fade-end?` When non-nil, causes the end of the scrollable view to fade out.
-  - `on-scroll` Callback called on the on-scroll event of the FlatList. Only
-    used when `scrollable?` is non-nil.
-  - `scrollable?` When non-nil, use a scrollable flat-list to render tabs.
-  - `scroll-on-press?` When non-nil, clicking on a tag centers it the middle
-    (with animation enabled).
-  "
+(defn view-internal
```

Comment:
added a malli schema and removed doc string

Reply:
`defn- view-internal`